.travis.yml

```diff
@@ -43,7 +43,7 @@ env:
   # - DH_REPO="cloudsuite/spark" IMG_TAG="latest" DF_PATH="./commons/spark/2.3.1"
   # - DH_REPO="cloudsuite/spark" IMG_TAG="1.5.1" DF_PATH="./commons/spark/1.5.1"
   # - DH_REPO="cloudsuite/spark" IMG_TAG="2.1.0" DF_PATH="./commons/spark/2.1.0"
-  - DH_REPO="cloudsuite/spark" IMG_TAG="2.3.1" DF_PATH="./commons/spark/2.3.1" DBX_PLATFORM=linux/amd64
+  - DH_REPO="cloudsuite/spark" IMG_TAG="2.4.5" DF_PATH="./commons/spark/2.4.5" DBX_PLATFORM=linux/amd64,linux/arm64,linux/riscv64
 
```

Comment:
I think travis entry for graph-analytics and in-memory analytics 4.0 should be added.

Reply:
Updated